blockchain/state/doc.go

```diff
@@ -18,5 +18,26 @@
 // This file is derived from core/state/statdb.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package state provides a caching layer atop the Klaytn state trie.
+/*
+Package state provides a caching layer atop the Klaytn state trie.
+This package is used to read/write stateObject from/to StateDB and it also acts as a bridge between
+the objects and the persistent layer.
+
+Overview of state package
+
+stateObject represents a Klaytn account which is being modified. It is identified by its address.
+Once it is loaded from the persistent layer, it is cached and managed by StateDB.
+StateDB manages stateObjects and operations to them. Most of the operations are done by StateDB,
```

Comment:
`StateDB manages stateObjects and operations to them.` this sentence has almost the same meaning with the above line.

Reply:
@aidan-kwon It is, but this seems to be needed. Please share your opinion if you have better expressions =)

Reply:
I resolve this conversation because @KimKyungup gave us a better idea.